REQUIRE

```diff
@@ -4,7 +4,7 @@ Distributions
 Docile
 DualNumbers
 FITSIO
-ForwardDiff 0.0.2 0.0.3
+ForwardDiff 0.0.2 0.0.3+
```

Comment:
It's stable, they're on release candidate 2 now. I think we just need one more change to `PriorParams` to make Celeste compatible with 0.4